packages/astro/test/units/assets/fonts/load.test.js

```diff
@@ -82,4 +90,10 @@ it('loadFonts()', async () => {
 	assert.equal(Array.from(hashToUrlMap.keys()).length > 0, true);
 	assert.deepStrictEqual(Array.from(resolvedMap.keys()), ['Roboto']);
 	assert.deepStrictEqual(logs, ['Fonts initialized']);
+	assert.equal(
+		resolvedMap
+			.get('Roboto')
+			.css.includes(':root { --astro-font-Roboto: Roboto, "Roboto fallback: Arial", sans-serif; }'),
```

Comment:
Should `--astro-font-Roboto` be `--astro-font-roboto`?

Reply:
No it's correct in this case because it'm not using `kebab` in the test, i'm passing the name through